github4s/src/main/scala/github4s/domain/Issue.scala

```diff
@@ -30,31 +29,32 @@ final case class Issue(
     html_url: String,
     number: Int,
     state: String,
-    user: Option[User],
-    assignee: Option[User],
     labels: List[Label] = List.empty,
-    locked: Option[Boolean],
     comments: Int,
-    pull_request: Option[IssuePullRequest],
-    closed_at: Option[String],
     created_at: String,
-    updated_at: String
+    updated_at: String,
+    body: Option[String] = None,
+    user: Option[User] = None,
+    assignee: Option[User] = None,
+    locked: Option[Boolean] = None,
+    pull_request: Option[IssuePullRequest] = None,
+    closed_at: Option[String] = None
 )
 
 final case class Label(
-    id: Option[Long],
     name: String,
-    description: Option[String],
-    url: Option[String],
     color: String,
-    default: Option[Boolean]
+    id: Option[Long] = None,
+    description: Option[String] = None,
+    url: Option[String] = None,
+    default: Option[Boolean] = None
 )
 
 final case class IssuePullRequest(
-    url: Option[String],
-    html_url: Option[String],
-    diff_url: Option[String],
-    patch_url: Option[String]
+    url: String,
+    html_url: String,
+    diff_url: String,
+    patch_url: String
```

Comment:
it's highly likely that the options were there for a reason.

Reply:
`IssuePullRequest` used in one place `Issue` where it's already `Option` field. I have check the [Issue docs](https://docs.github.com/en/rest/reference/issues) and don't see cases with some of this fields.

Well, I can change it back, but looks like overhead.

Reply:
Unfortunately, the documentation is not a reliable source of truth. we've been bit by issues like this in the past.

Reply:
Sure, return option